### PR TITLE
Gpl976 all other events

### PR DESCRIPTION
**Pull request by emrojo**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: lighthouse/classes/events/biosero/source_all_negatives.py [hunk 1]
```diff
@@ -1,29 +1,49 @@
 import logging
```
Comment (andrewsparkes) [inline lighthouse/classes/events/biosero/source_all_negatives.py:1]: This class and file not needed, source_no_pickable_samples is what we called this event for biosero.

File: tests/endpoints/events/biosero/test_source_all_negatives.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
+from http import HTTPStatus
```
Comment (andrewsparkes) [inline tests/endpoints/events/biosero/test_source_all_negatives.py:1]: this file can go too, event source_no_pickable_samples is the one we will use

**Review by andrewsparkes — CHANGES_REQUESTED** [review]
All I noticed was the extra event type we don't need, 'source_all_negatives', that is replaced by 'source_no_pickable_samples' on biosero.

File: lighthouse/classes/events/biosero/source_all_negatives.py (hunk 1 above)
Comment (emrojo) [reply lighthouse/classes/events/biosero/source_all_negatives.py:1]: ok, is removed in the code reviewed branch

File: tests/endpoints/events/biosero/test_source_all_negatives.py (hunk 2 above)
Comment (emrojo) [reply tests/endpoints/events/biosero/test_source_all_negatives.py:1]: ok

**Pull request closed without merging**